sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -125,6 +126,75 @@ def test_target_overhead(self):
         clock.sleep(batch_duration(actual_sizes[-1]))
     self.assertEqual(expected_sizes, actual_sizes)
 
+  def test_variance(self):
+    clock = FakeClock()
```

Comment:
How about we fix the initial start time of FakeClock() to reduce potential for flakiness?

Reply:
Actually, it shouldn't matter.